pupil_src/shared_modules/video_capture/base_backend.py

```diff
@@ -199,10 +199,26 @@ def replace_backend_manager(manager_class):
 class Playback_Source(Base_Source):
     allowed_speeds = [.25, .5, 1., 1.5, 2., 4.]
 
-    def __init__(self, g_pool, timed_playback=False, playback_speed=1., *args, **kwargs):
+    def __init__(self, g_pool, stand_alone=True, timed_playback=False, playback_speed=1., *args, **kwargs):
+        '''
+        Possible configurations:
+                                    stand_alone
+                             |   True    | False
+                       ------+-----------+--------
+                        True |  Capture  |   /
```

Comment:
why not make one var:

playback_speed

possible values: 
a) a float for actual playback speed
b) "external" for use in player
c) "free_running"

Reply:
You could argue that there is no need to change playback speed for `Playback_Source`'s in Capture. Therefore we could move the `playback_speed` field to `Seek_Control` and merge `stand_alone` + `timed_playback` into one `timing` (?) field that allows one of the following values:
- `'external`, currently `stand_alone=False, timed_playback=False`, use case: Player
- `'own`, currently  `stand_alone=True, timed_playback=True`, use case: Capture source
- `None`

Reply:
I like this. lets do it like this.

Reply:
@papr  can we still add this sugenstion? I like you solution